packages/mui-material-next/src/styles/shape.ts

```diff
@@ -0,0 +1,19 @@
+import { MD3Shape } from './Theme.types';
+
+const mdSysShape: MD3Shape = {
+  corner: {
+    none: '0',
```

Comment:
```suggestion
    none: '0px',
```
Let's use a valid CSS unit.